### PR TITLE
TML-33691 Create FileManager SDK

### PR DESCRIPTION
**Pull request by mmirboboev**

### CODE REVIEW

**Review by umedjan — APPROVED** [review]

File: sellercenter-filemanger/src/main/java/com/gfg/sellercenter/filemanager/request/SearchFilesRequest.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        if (getCreatedAtLte() != null) {
+            map.put("createdAtLte", getCreatedAtLte().toString());
+        }
+
+        if (getKind() != null) {
+            map.put("kind", getKind().toString());
+        }
+
+        if (getStatuses() != null) {
+            map.put("statuses", getKind().toString());
```
Comment (Cloudwalker1986) [inline sellercenter-filemanger/src/main/java/com/gfg/sellercenter/filemanager/request/SearchFilesRequest.java:52]: get kind?

**Review by Cloudwalker1986 — CHANGES_REQUESTED** [review]
Except for the one comment all is fine.

File: sellercenter-filemanger/src/main/java/com/gfg/sellercenter/filemanager/request/SearchFilesRequest.java (hunk 1 above)
Comment (mmirboboev) [reply sellercenter-filemanger/src/main/java/com/gfg/sellercenter/filemanager/request/SearchFilesRequest.java:52]: Oh thanks for noticing.

**Review by Cloudwalker1986 — APPROVED** [review]

File: sellercenter-filemanger/pom.xml [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    <version>1.0-SNAPSHOT</version>
+
+    <name>sellercenter-filemanger</name>
+    <url>https://github.com/GFG/java-packages</url>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <java.version>11</java.version>
```
Comment (yevgeniyskulditskiy) [inline sellercenter-filemanger/pom.xml:18]: I never had a time to read how actually jar files compatible to each other. So if someone is using java 8, will it be possible to include java 11 jar library?

**Review by yevgeniyskulditskiy — APPROVED** [review]

**Pull request merged**